SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -646,7 +646,10 @@ void CFVMFlowSolverBase<V, R>::ComputeVorticityAndStrainMag(const CConfig& confi
     const su2double vy = nodes->GetVelocity(iPoint, 1);
     const su2double y = geometry->nodes->GetCoord(iPoint, 1);
     AD::StartPreacc();
+    AD::SetPreaccIn(omegaMax);
+    AD::SetPreaccIn(strainMax);
     AD::SetPreaccIn(VelocityGradient, nDim, nDim);
+    AD::SetPreaccIn(Vorticity, 3);
```

Comment:
The tape check detects an issue, but is it really an issue in normal AD modes? Can we put the extra preaccumulation behind some #ifdef ADJOINT_DEBUG so that we still get memory savings when possible?

Reply:
I think it's really an issue;  either the computation of `omegaMax` and `strainMax` shall be on the tape or not. If we exclude their computation from the taping, a regression test fails, as I said. If we want them in the recording, and the in preaccumulation, we have do it correctly, I think. So, what do we want?